src/simulator/ackermann_model.cpp

```diff
@@ -0,0 +1,96 @@
+#include "simulator/ackermann_model.h"
+#include "config_reader/config_reader.h"
+#include "shared/util/timer.h"
+#include "shared/math/math_util.h"
+
+using std::string;
+using f1tenth_simulator::AckermannCurvatureDriveMsg;
+using math_util::AngleDiff;
+using math_util::AngleMod;
+
+namespace ackermann {
```

Comment:
It's worth considering wrapping all the drives in a single outer namespace such as `drive`.

Reply:
Will consider when we have a second motion model. Want to see if there are any issues or gains from this.